UPGRADE-4.0.md

```diff
@@ -200,6 +200,7 @@ Filesystem
  * The `Symfony\Component\Filesystem\LockHandler` has been removed,
    use the `Symfony\Component\Lock\Store\FlockStore` class
    or  the `Symfony\Component\Lock\Store\FlockStore\SemaphoreStore` class directly instead.
+ * support for passing relative paths to `Filesystem::makePathRelative()` has been removed in 4.0
```

Comment:
First letter should be uppercase for consistency, same for UPGRADE-3.3.  `in 4.0` could be removed also